source/documentation/deploying_services/postgresql.md

```diff
@@ -271,6 +271,93 @@ cf update-service SERVICE_NAME -c '{"reboot": true, "force_failover": true}'
 
+### Add or remove extensions for a Postgresql service instance
+
+GOV.UK PaaS supports and enables the following extensions by default:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|uuid-ossp|9.5, 10|
+|postgis|9.5, 10|
+|citext|10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+GOV.UK PaaS supports the following extensions, but they are not enabled by default:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|pg_stat_statements|9.5, 10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+You can add or remove extensions for a PostgreSQL service instance by:
+
+- creating a new service instance
+- updating an existing service instance
+- restoring a PostgreSQL service instance snapshot
+
+#### Create a new service instance
+
+You [create a new PostgreSQL service instance](#set-up-a-postgresql-service) by running `cf create-service`. You can enable extensions in this new service instance by running:
+
+```
+cf create-service SERVICE_NAME -c '{"enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+
+where:
+
+- `SERVICE_NAME` is a unique descriptive name for this service instance
+- `EXTENSION_1...N` are the names of the extensions you want to enable
+
+#### Update an existing service instance
+
+You add or remove extensions on an existing PostgreSQL service instance by updating your service with the new list of extensions:
+
+```
+cf update-service SERVICE_NAME -c '{"enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+
+For example, your PostgreSQL service instance is named `my-pg-service` and has 3 extensions enabled, `uuid-ossp`, `postgis` and `citext`. Run the following to remove `citext`:
+
+```
+cf update-service my-pg-service -c '{"enabled_extensions": ["uuid-ossp","postgis"]}'
+```
+
+If you add the `pg_stat_statements` extension to your service instance, this extension will not start until the GOV.UK PaaS team reboots your service instance.
+
+This reboot will cause some service downtime. You should schedule this downtime to minimise service disruption.
+
+#### Restore a service instance
+
+If you have a [paid PostgreSQL plan](#paid-plans-postgresql), the Amazon RDS system [backs up your PostgreSQL service data](#postgresql-service-backup) every night. This includes all extensions that were enabled at time of backup.
+
+When you [restore a PostgreSQL service instance from a snapshot](#restoring-a-postgresql-service-snapshot), that restored service instance will by default include these extensions.
+
+If the snapshot contains extensions that GOV.UK PaaS no longer supports, the restored service instance will still include these extensions. However, you cannot create new service instances with unsupported extensions.
```

Comment:
I'm not sure whether we should include this as this is theoretical and we don't know whether it's true or not. I would remove it.

Reply:
The first paragraph is true as it stands. The second paragraph I agree is hypothetical, since we've not stopped supporting any extensions to date.

Reply:
(I referred only to the second paragraph)

Reply:
removing